index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+ <div id="main"> 
+  <nav>
+    <div>
+      <h1 id="title">Bjarne Stroustrup</h1>
+      <h3>He is the creator of C++ programming language</h3>
+    </div>
+  </nav>
+  <section class="images-section">
+    <div class="images" id="img-div">
+      <img id="image" src="images/Bjarne_Stroustrup_UC3M_2-1.jpg" alt="">
```

Comment:
Images must have the ```alt``` attribute with a small description of the image.
Remember that the content of this attribute is going to be displayed if your image cannot be displayed for some reason.